### PR TITLE
Add skipped collections to MongoDB

### PR DESCRIPTION
**Pull request by kbariotis**
I have also updated MongoDB driver due to incompatibilities with Mongo 3.2. A major release should occur IMO. Tests refactored, started a discussion in #42.

Tests also doesn't suppose that the `system.indexes` collection will be remove, since you [can't delete a `system.` namespace collection](https://docs.mongodb.org/manual/reference/system-collections/).

### CODE REVIEW

**Comment by kbariotis** [comment]
Hmm it seems like it did delete the `system.indexes` collection. It could be because of the Mongo version of Travis.

File: config/cleaner-config.js [hunk 1]
```diff
@@ -5,5 +5,9 @@ module.exports = {
 
   postgresql: {
     skipTables: ['schema_migrations']
+  },
+
+  mongodb: {
+      skipTables: ['schema_migrations']
```
Comment (emerleite) [inline config/cleaner-config.js]: Should be skipCollections instead of skipTables, do you agree?

**Comment by emerleite** [comment]
Thanks for the PR @kbariotis. Please, just make the few adjustments and make sure all tests pass and then I'll integrate your PR.

Cheers.

**Comment by kbariotis** [comment]
done @emerleite. Thank you.

**Comment by emerleite** [comment]
Released version 1.0.0 - @kbariotis

**Pull request merged**